website/static/js/addProjectPlugin.js

```diff
@@ -82,7 +86,7 @@ var AddProject = {
             var data;
             self.viewState('processing');
             if(self.options.parentID) {
-                url = $osf.apiV2Url('nodes/' + self.options.parentID + '/children/', { query : {}});
+                url = $osf.apiV2Url('nodes/' + self.options.parentID + '/children/', { query : {'inherit_contributors' : self.newProjectInheritContribs()}});
```

Comment:
See https://github.com/CenterForOpenScience/osf.io/pull/5884/files#r68627316 . I think this will result in contributors being inherited, no matter what value the user inputs here.